.travis.yml

```diff
@@ -16,6 +16,16 @@
 #
 jobs:
   include:
+    - language: java
+      python: 3.6
+      env:
+        - TOXENV=license-check
+        - TRAVIS_CACHE=$HOME/.travis_cache/
+      jdk:
+        - openjdk8
+      install:
+        - sudo pip install --upgrade pip
+        - sudo pip install codecov tox
```

Comment:
I don't think `codecov` is needed here.

Reply:
Sorry I had missed that and merged from mobile after @dpgaspar stacked commits that fixed the build on my PR...